default.py

```diff
@@ -264,17 +264,23 @@ def getEpisodeTVDB(self, xbmc_id):
         rpccmd = {'jsonrpc': '2.0', 'method': 'VideoLibrary.GetEpisodeDetails', 'params': {"episodeid": int(xbmc_id), 'properties': ['season', 'episode', 'tvshowid', 'showtitle', 'uniqueid']}, 'id': 1}
         rpccmd = json.dumps(rpccmd)
         result = xbmc.executeJSONRPC(rpccmd)
-        result = json.loads(result)        
-        log('result=%s' % result)    
-        log('episode_id=%s' % result['result']['episodedetails']['uniqueid']['unknown'])
-        
+        result = json.loads(result)
+        log('result=%s' % result)
+        if 'unknown' in result['result']['episodedetails']['uniqueid']:
+            episode_id = result['result']['episodedetails']['uniqueid']['unknown']
+        if 'tvdb' in result['result']['episodedetails']['uniqueid']:
+            episode_id = result['result']['episodedetails']['uniqueid']['tvdb']
+        else:
```

Comment:
episode_id = result['result']['episodedetails']['uniqueid']['imdb']  may also exist. You should add an extra if for that.